lms/djangoapps/grades/tests/test_tasks.py

```diff
@@ -202,7 +202,7 @@ def test_persistent_grades_not_enabled_on_course(self, default_store):
             self.set_up_course(enable_persistent_grades=False)
             self.assertFalse(PersistentGradesEnabledFlag.feature_enabled(self.course.id))
             with check_mongo_calls(0):
-                with self.assertNumQueries(0):
+                with self.assertNumQueries(1):
```

Comment:
I don't think this should be a blocker, as the additional load would be one query per task, performed onon the task worker, and I don't believe it can introduce any sort of database contention, as it is performed as a single update query, along the lines of:

```sql
UPDATE celery_utils_failedtask SET datetime_resolved = NOW() WHERE task_id = ? and datetime_resolved IS NONE;

```
If another worker tries to update the same table, there won't be an integrity error, as no unique indexes are modified.  One of the workers will just win, and it doesn't really matter which: the datetime_resolved column will be set to a non-null timestamp around the right time.